lib/config.rb

```diff
@@ -15,13 +15,15 @@ module Config
   # Ensures the setup only gets run once
   @@_ran_once = false
 
-  mattr_accessor :const_name, :use_env, :env_prefix, :env_separator, :env_converter, :env_parse_values
+  mattr_accessor :const_name, :use_env, :env_prefix, :env_separator,
+                 :env_converter, :env_parse_values, :fail_on_missing
   @@const_name = 'Settings'
   @@use_env    = false
   @@env_prefix = @@const_name
   @@env_separator = '.'
   @@env_converter = :downcase
   @@env_parse_values = true
+  @fail_on_missing = false
```

Comment:
Shouldn't we use `@@` as for every other setting?

Reply:
I definitely agree, I changed it to `@` because Code Climate was complaining about it on my first version of this PR. If you are okay with using this class variable and make Code Climate upset about it, then I will rollback to `@@fail_on_missing`

Reply:
I think that’s better to keep consistency. Would you agree?

Reply:
I updated the code to keep consistency.